deltacat/compute/compactor_v2/constants.py

```diff
@@ -35,3 +35,8 @@
 
 # Whether to drop duplicates during merge.
 DROP_DUPLICATES = True
+
+# PARQUET to PYARROW inflation multiplier
+# This is the observed upper bound inflation for parquet
+# size in metadata to pyarrow table size.
+PARQUET_TO_PYARROW_INFLATION = 4
```

Comment:
This PARQUET_TO_PYARROW_INFLATION is specific to reading parquet file?

Reply:
Yes.